photon-ml/src/integTest/scala/com/linkedin/photon/ml/FeatureIndexingJobTest.scala

```diff
@@ -68,16 +73,16 @@ class FeatureIndexingJobTest {
     for (partitionNum <- 1 to 2) {
       testShardedJob(
         tempFile.toString, partitionNum, addIntercept = true, featureShardIdToFeatureSectionKeysMap,
-        featureShardIdToExpectedDimension)
+        featureShardIdToInterceptMap, featureShardIdToExpectedDimension)
       testShardedJob(
         tempFile.toString, partitionNum, addIntercept = false, featureShardIdToFeatureSectionKeysMap,
-        featureShardIdToExpectedDimension)
+        featureShardIdToInterceptMap, featureShardIdToExpectedDimension)
       testShardedJob(
         tempFile.toString, partitionNum, addIntercept = true, featureShardIdToFeatureSectionKeysMap,
-        featureShardIdToExpectedDimension)
+        featureShardIdToInterceptMap, featureShardIdToExpectedDimension)
       testShardedJob(
         tempFile.toString, partitionNum, addIntercept = false, featureShardIdToFeatureSectionKeysMap,
-        featureShardIdToExpectedDimension)
+        featureShardIdToInterceptMap, featureShardIdToExpectedDimension)
```

Comment:
Scala style: If all of the arguments cannot fit on one line, they should each be on a separate line